client/src/App.jsx

```diff
@@ -1,18 +1,62 @@
-import React from 'react';
+import React, { Component } from 'react';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import Login from './components/Login';
 import './App.css';
 import NewTicket from './components/NewTicket';
 import MainSidebar from './components/MainSidebar';
 import WrappedTicket from './components/WrappedTicket';
 
-export default () => (
-  <BrowserRouter>
-    <Switch>
-      <Route exact path="/"/>
-      <Route exact path="/new-ticket" component={NewTicket} />
-      <Route exact path="/login" component={Login} />
-      <Route exact path="/main-sidebar" component={() => <MainSidebar selected="tickets" />} />
-    </Switch>
-  </BrowserRouter>
-);
+class App extends Component {
+  state = {
+    wrappedTickets: [],
+  }
+
+  componentDidMount() {
+    this.setState({
+      wrappedTickets: [{
```

Comment:
@Jamalat-shamallakh Could you move this wrappedTickets data into the initial state (up on line 11) instead of calling setState() inside componentDidMount?

If possible, try to avoid calling setState in componentDidMount (.. it's not always easy to do this though).  componentDidMount means that the component just finished rendering, and setState will cause the component to re-render.  So it's an extra render that can be avoided if that state was already available for the first render.